### ISSUE TITLE
Docker container does not correctly load AWS profile

### ISSUE THREAD

**Issue by daaru00** [disneystreaming/ssm-helpers#11]
Hi,

I'm using the Docker containers setup, when I run "ssm-session":
```
docker run -it --rm -v $HOME/.aws:/root/.aws -e AWS_PROFILE=personal \
    -e AWS_REGION=eu-west-1 \
    docker.pkg.github.com/disneystreaming/ssm-helpers/ssm-session
```
appear this error:
```
ERROR NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
```

Credentials are correct (I using it every day) but neither if using a simple profile:
```
[personal]
aws_access_key_id = xxxxxxxxxxxxxxxxxxxx
aws_secret_access_key = xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
or an assume roles:
```
[test]
role_arn = arn:aws:iam::000000000:role/MyAssumeRole
source_profile = personal
region = eu-west-1
```
both does not works.

**Comment by rothgar** [comment]
Interesting you get an error. I have a very similar setup with assumed roles and am able to use the container.

Can you try running `aws` inside the container directly to see if you also get an error?

```
docker run -it --rm -v $HOME/.aws:/root/.aws -e AWS_PROFILE=personal \
    -e AWS_REGION=eu-west-1 --entrypoint=bash \
    docker.pkg.github.com/disneystreaming/ssm-helpers/ssm-session
```
I realized there's an error with the container missing the python-six package right now so make sure you `docker pull docker.pkg.github.com/disneystreaming/ssm-helpers/ssm-session` first.

Once you have a shell in the container try running `aws ec2 describe-instances` and let me know if you get an error. If you **don't** get an error try running `ssm-session` and see if you get the same error.

**Comment by daaru00** [comment]
Hi @rothgar,

aah I got it, instead of directly using `~/.aws/credentials` I'm using a link to `~/Private/aws/credentials` where `~/Private` is a directory encrypted to the disk ([here](https://help.ubuntu.com/community/EncryptedPrivateDirectory) more info about the setup).

Obviously mounting `~/.aws` folder didn't work for me, I changed volume mount point to:
```bash
docker run -it --rm ~/Private/aws/:/root/.aws -e AWS_PROFILE=test -e AWS_REGION=eu-west-1 docker.pkg.github.com/disneystreaming/ssm-helpers/ssm-session
```
and it works!

It just depends by my own environment, I think the issue can be closed.
